### ISSUE TITLE
File with split audio track crashes on export, takes very long to load

### ISSUE THREAD

**Issue by Esn024** [OpenShot/openshot-qt#1520]
**System Details:**
* Operating System / Distro: Windows 7 Pro SP1
* OpenShot Version: 2.4.1-dev1 (launch.exe is dated April 14, 2018)

**Issue Description and steps to reproduce:**
So, I have some video footage in which the camera started filming a musical performance a little too late, and was accidentally turned off for 40 seconds in the middle. Thankfully, I also made a separate audio recording (a .m4a file) capturing the entire time period. So what I tried to do in OpenShot is line up the two video clips to the right place in the audio (using a few photos to fill the missing time), then I split the long audio track into 4 parts (at the exact spots where the two video recordings start/end), set the volume for the ones where there's no video to 100%, and where there IS video (which has its own audio track) to 0%.

The export crashes when it gets to the final one of the 4 audio sections (I think at either 86.7% or 89.95?)). At each previous "audio split" point, it seems stuck for a little while, but eventually moves past it. If I do a partial export of just the first 500 frames of the video with the first two audio sections, it manages to complete it.

Also, when I opened my project file again, it took a really long time to load. It takes MUCH longer to load after I have split the audio file than if the audio is one long unsplit track.

Is there any way to work around the issue? For example, can I set the volume of a long audio file to be 0% or 100% at specific points in the middle (not just a fade in/out at the beginning & end), without splitting it? Thank you very much. Sorry if I misunderstood something about how the program works...

[libopenshot.log](https://github.com/OpenShot/openshot-qt/files/1930297/libopenshot.log)
[openshot-qt.log](https://github.com/OpenShot/openshot-qt/files/1930298/openshot-qt.log)

**Comment by Esn024** [comment]
Strange... I just tried it again and it didn't crash this time when I tried to export (although it did take quite a while to load...). The only thing I changed is that I made double-sure that the very last split audio section was set to volume=0% (I may have forgotten to do that the first time, so possibly both the video clip's audio and the .m4a file were both set to 100%).

I'm attaching the updated log files, just in case they're of any use. I'm sorry if I wasted anyone's time, although it's still strange that the program crashed the first time (and it's not the first time something similar has happened).

I WOULD still be very grateful if anyone has an answer to the question in the final paragraph of my previous post.

[libopenshot.log](https://github.com/OpenShot/openshot-qt/files/1930354/libopenshot.log)
[openshot-qt.log](https://github.com/OpenShot/openshot-qt/files/1930355/openshot-qt.log)
